canonical-web-and-design/ubuntu.com
# Remove /16-04/gcp page

## Done

- Remove the /gcp/16-04 page for now
- Added 16-04 to the footer

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/16-04/gcp 
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the page is removed and that 16-04 is in the footer.


## Issue / Card

Fixes #9416
